atom-jobs/build-common.groovy

```diff
@@ -523,6 +519,20 @@ if [ ${OS} == 'darwin' ]; then
     ls -l ./release-darwin/tiflash/
     mv release-darwin ${TARGET}
 else
+    if [[ "${USE_TIFLASH_RUST_CACHE}" == "true" ]]; then
```

Comment:
USE_TIFLASH_RUST_CACHE us a booleanParam,  properties need set this param

Reply:
fixed